lib/Backend/Opnd.cpp

```diff
@@ -2288,6 +2285,35 @@ IndirOpnd::New(RegOpnd *baseOpnd, RegOpnd *indexOpnd, byte scale, IRType type, F
 ///----------------------------------------------------------------------------
 
 IndirOpnd *
+IndirOpnd::New(RegOpnd *indexOpnd, int32 offset, byte scale, IRType type, Func *func)
+{
+    IndirOpnd * indirOpnd;
+
+    indirOpnd = JitAnew(func->m_alloc, IR::IndirOpnd);
```

Comment:
>IR:: [](start = 39, length = 4)

nit: IR:: prefix is unnecessary. Add it might cause some confuse because `InDirOpnd` is the current class.